servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -271,6 +271,12 @@ bool RenderForwardMobile::free(RID p_rid) {
 
 /* Render functions */
 
+float RenderForwardMobile::_render_buffers_get_color_scale() {
```

Comment:
I have the feeling this should be called luminance_scale rather than color_scale

Reply:
or luminance_multiplier?

Reply:
I'm open for either

Reply:
Settled on `luminance_multiplier`